lib/web/apiserver.go

```diff
@@ -3623,3 +3624,11 @@ func (h *Handler) authExportPublic(w http.ResponseWriter, r *http.Request, p htt
 	// It also handles the Range negotiation
 	http.ServeContent(w, r, "authorized_hosts.txt", time.Now(), reader)
 }
+
+// getIsDashboard returns a bool indicating if the cluster is a
+// dashboard cluster.
+// Dashboards are clusters running on cloud infrastructure that aren't
+// Teleport Cloud users
```

Comment:
we should change to something like
"Dashboards are clusters running on cloud infrastructure that aren't tied to a Teleport Cloud user". 
I'm unfamiliar with our terminology here, but if we consider a cluster a 'user' in this sense, then ignore my comment.

Reply:
Yeah, that did sound correct. Updated to `Dashboard is a cluster running on cloud infrastructure that isn't a Teleport Cloud cluster`, what do you think?

Reply:
Sounds good to me!